### ISSUE TITLE
rmh.ppm not working with fitted Lennard-Jones model

### ISSUE THREAD

**Issue by dschuhmacher** [spatstat/spatstat#48]
There appears to be a bug in translating the parameters from the ppm-object to the rmhmodel-object for the Lennard-Jones process. 

```
set.seed(10)
xi <- rpoispp(50)
model <- ppm(xi~1, LennardJones(), rbord=0.1)
rmhmodel(model)
```
This throws the error `‘epsilon’ should be a single number`. It seems the culprits are lines 24/25 in rmhmodel.ppm.R. The parameter `sigma` for rmh is taken by partial matching from the "irregular parameter" `sigma0` of `model` (which already is not correct) and the parameter `epsilon` gets assigned NULL because no such partial match exists.

**Comment by baddstats** [comment]
Thank you! This bug will be fixed shortly [and will be included in the public release of `spatstat 1.50-0` in the next few days.]

**Comment by baddstats** [comment]
Fixed now.
